examples/pubsub/subscriber.py

```diff
@@ -45,23 +45,23 @@ def main():
     ctx = zmq.Context()
     s = ctx.socket(zmq.SUB)
     s.connect(connect_to)
-    s.setsockopt(zmq.SUBSCRIBE, '')
+    s.setsockopt(zmq.SUBSCRIBE, ''.encode('utf-8'))
 
     sync(connect_to)
 
-    start = time.clock()
+    start = time.process_time()
```

Comment:
`clock()` was removed since `python 3.3` and replaced it with `process_time()` and `perf_counter()`